code/modules/client/loadout/loadout_accessories.dm

```diff
@@ -184,3 +192,40 @@
 	display_name = "black bishop's cloak"
 	path = /obj/item/clothing/neck/cloak/chap/bishop/black
 	allowed_roles = list("Chaplain")
+
+//Masks
+
+/datum/gear/accessory/mask
+	subtype_path = /datum/gear/accessory/mask
+	cost = 1500
+
+/datum/gear/accessory/mask/clown
+	display_name = "clown mask"
+	description = "Dress up just like your hero! The... clown...?"
+	path = /obj/item/clothing/mask/gas/clown_hat
+
+/datum/gear/accessory/mask/mime
+	display_name = "mime mask"
+	description = "..."
+	path = /obj/item/clothing/mask/gas/mime
+
+/datum/gear/accessory/mask/russian_balaclava
+	display_name = "russian balaclava"
+	description = "cyka blyat!"
```

Comment:
As funny as it is, can't have this in the codebase

Reply:
Is it because of the swearsies or because it's a balaclava? Just for clarification.